core/src/main/java/org/elasticsearch/action/update/UpdateRequest.java

```diff
@@ -893,4 +894,31 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.endObject();
         return builder;
     }
+
+    @Override
+    public String toString() {
+        String res = "update {[" + index + "][" + type + "][" + id + "]";
+        if (docAsUpsert) {
+            res += (", doc_as_upsert[" + docAsUpsert + "]");
+        }
+        if (doc != null) {
+            res += (", doc[" + doc + "]");
+        }
+        if (script != null) {
+            res += (", script[" + script + "]");
+        }
+        if (upsertRequest != null) {
+            res += (", upsert[" + upsertRequest + "]");
+        }
+        if (scriptedUpsert) {
+            res += (", scripted_upsert[" + scriptedUpsert + "]");
+        }
+        if (detectNoop) {
```

Comment:
This is `boolean` - I think it'd be better to include it in the output whether it's `true` or `false`.

Reply:
Fixed. One more field which is a boolean and not listed here is also updated.